conduct.md

```diff
@@ -7,16 +7,282 @@
+objectively analyzed the options for the best of the team. Trust and respect
+the decision made in order to move on the project.
+
+* **Unacceptable (U):**
+  * Making decisions without notifying team members.
+  * Question the decision after it is made.
+  * Making a decision based on only popular opinion.
+
+## Postmortems
+
+In the postmortems process of the project we will use proper communication and
+teamwork in order to solve problems and issues. Going through postmortems will
+help us to debug and ensure problems that arise get taken care of smoothly and
+surely.
+
+* **Acceptable Postmortems (C)**
+  * Talk in the first part of lab section to discuss where we are in the project.
+  * Team leaders can lead through this discussion.
+  * Teams will bring up problems and errors that occurred, or what worked well.
+  * After finishing a project teams should be assigned to look after the program.
+  * Work as a team to fix problems.
+  * Use proper communication to talk to one another.
+
+* **Unacceptable (U)**
+  * Do not blame anyone for errors or problems
+  * Do not work alone use your team members as a guide
+  * Do not attempt to change things without talking to others
+
+## Conflicts
+
+In the conflicts section of the project, we deal with conflicts in a
+professional manner by communicating any type of issue/ conflict with any of the
+team leaders, technical leads, or the CEO directly.
+
+* **Acceptable Behavior (C)**
+  * Team Members shall:
+    * Not unduly use their position/ influence in this project for personal
+    benefit or to benefit relatives or close associates.
+    * Report a conflict or potential conflict with either a team leader,
+    technical lead or the CEO directly as soon as it arises or at the nearest
+    availability.
+  * Team Leads shall:
+    * Be open and respect both sides looking for the most beneficial option
+    towards the project.
+    * Make decisions with the best interest of the team as a whole upon proper
+    discussion.
+* **Unacceptable Behavior (U)**
+  * Team Members shall NOT:
+    * Bring about personal interests that will bring about conflicts with
+    those of the team as a whole.
+    * Move forward on a conflict that has not yet been discussed or resolved
+    to prevent even further conflicts to arise.
+  * Team Leads shall NOT:
+    * Disregard any conflict that has been brought to their attention.
+    * Use their position/ influence to make any type of decision that is not
+    beneficial to the team as a whole.
+
+## Infractions
+
+An ***infraction*** shall hereby be defined as a penalty given as a result of a
+violation of the above outlined Conduct Guide by any individual who has agreed
+to adhere to the Conduct Guide via digital signature below. The infractions for
+any individual shall be recorded and managed by the instructor.
+
+A *violation* of the Conduct Guide is defined as any behavior that results in receiving
```

Comment:
Who determines the "receiving" of a U? Since a Level I infraction is seemingly the same thing, this should perhaps be reworded to say "committing an action that falls into the U category" or something similar.

Reply:
From the `conduct.md` (emboldened lines for emphasis):

> A Level I infraction is defined as a **formally documented violation** of the above outlined Conduct Guide. A Level I infraction may be suggested to the instructor by a team or project leader for any given project.

The intent is that a *violation* is an instance where a team member behaves in a way that can be defined as Unacceptable (U), whereas an *infraction* is simply the formal documentation of the violation with the instructor. Would changing the wording for a violation to something along the lines of: 

> A *violation* of the Conduct Guide is defined as any behavior that can be considered Unacceptable (U) as outlined above.

Be a more valid approach in your opinion?

Reply:
Definitely, since that clearly explains what a violation is. Before, it was unclear whether a violation was something someone gave, or something someone did.

When you feel as though this issue has been resolved, @shafferz, feel free to mark this conversation as resolved.

Reply:
Alright, I went ahead and made some changes. I am currently awaiting approval for the Pull Request on Aubrey's fork, which should come shortly. 

Thanks for your suggestions, @Michionlion!